packages/plugins/integration-zapier-ui/src/lib/components/zapier/zapier.component.ts

```
@@ -0,0 +1,173 @@
+@Component({
+	selector: 'ngx-zapier',
+	templateUrl: './zapier.component.html',
+	styleUrls: ['./zapier.component.scss'],
+	standalone: false
+})
+export class ZapierComponent extends TranslationBaseComponent implements OnInit, OnDestroy {
+	public tabs: NbRouteTab[] = [];
+	public menus: NbMenuItem[] = [];
+	public integrationId: ID;
+	public organization: IOrganization;
+	public loading: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)